### PR TITLE
docs: updated users image part in README.md

### LINKED ISSUE #3709: update Slack invitaion link
# Improve Docs

## Please describe which part of docs should be improved or typo fixed

Hi, the Slack invitation link in the README.md[1] file is incorrect, it needs a special email address ends with apache.org.

[1] https://github.com/apache/apisix/edit/master/README.md

## Describe the solution you'd like

Use this invitation instead, it's my personal invitation link and will not expire.

Link: https://join.slack.com/t/the-asf/shared_invite/zt-lcbw4olf-_5s17RLU0Cx8xTvj2bNbjg

image

image

### LINKED ISSUE #3711: improve screenshots in docs
# Improve Docs

## Please describe which part of docs should be improved or typo fixed

Hi, I noticed that there have some screenshots from the v1 dashboard, does anyone would like to take some screenshots from the v2.x dashboard[2]?

[1] https://apisix.apache.org/docs/apisix/plugins/jwt-auth
[2] https://github.com/apache/apisix-dashboard

### PR DESCRIPTION
**Pull request by iamayushdas**
### What this PR does / why we need it:

resolve #3711 
resolve #3709 

### Pre-submission checklist:

* [ ] Did you explain what problem does this PR solve? Or what new features have been added?
* [ ] Have you added corresponding test cases?
* [x] Have you modified the corresponding document?
* [ ] Is this PR backward compatible? **If it is not backward compatible, please discuss on the [mailing list](https://github.com/apache/apisix/tree/master#community) first**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -40,7 +40,7 @@ The technical architecture of Apache APISIX:
 
 - Mailing List: Mail to dev-subscribe@apisix.apache.org, follow the reply to subscribe the mailing list.
 - QQ Group - 578997126
-- [Slack Workspace](http://s.apache.org/slack-invite) - join `#apisix` on our Slack to meet the team and ask questions
+- [Slack Workspace](https://join.slack.com/t/the-asf/shared_invite/zt-lcbw4olf-_5s17RLU0Cx8xTvj2bNbjg) - join `#apisix` on our Slack to meet the team and ask questions
```
Comment (juzhiyuan) [inline README.md:43]: Hi, thanks for this user wall! But we need to update 2 logos:

- DaoCloud: https://static.apiseven.com/logo/daocloud.png
- HelloTalk: https://static.apiseven.com/logo/HelloTalk.png

File: README.md [hunk 2]
```diff
@@ -319,7 +319,7 @@ Benchmark comparison test [details data](https://gist.github.com/membphis/137db9
 
 A wide variety of companies and organizations use APISIX for research, production and commercial product, including:
 
-<img src="https://raw.githubusercontent.com/api7/website-of-API7/master/user-wall.jpg" width="900" height="500" />
+<img src="https://user-images.githubusercontent.com/40708551/109481085-59834d80-7aa2-11eb-9181-fe0298aedb65.png" width="725" height="1700" />
```
Comment (juzhiyuan) [inline README.md:322]: Hi, thanks for this user wall! But we need to update 2 logos:

- DaoCloud: https://static.apiseven.com/logo/daocloud.png
- HelloTalk: https://static.apiseven.com/logo/HelloTalk.png

Comment (iamayushdas) [reply README.md:322]: > Hi, thanks for this user wall! But we need to update 2 logos:
> 
> * DaoCloud: https://static.apiseven.com/logo/daocloud.png
> * HelloTalk: https://static.apiseven.com/logo/HelloTalk.png

ok pushing that to my next commit

Comment (iamayushdas) [reply README.md:322]: please review the #3709

Comment (iamayushdas) [reply README.md:322]: > Hi, thanks for this user wall! But we need to update 2 logos:
> 
> * DaoCloud: https://static.apiseven.com/logo/daocloud.png
> * HelloTalk: https://static.apiseven.com/logo/HelloTalk.png

Pushing it to my next commit

Comment (iamayushdas) [reply README.md]: > Hi, thanks for this user wall! But we need to update 2 logos:
> 
> * DaoCloud: https://static.apiseven.com/logo/daocloud.png
> * HelloTalk: https://static.apiseven.com/logo/HelloTalk.png

@juzhiyuan check it has been fixed now

**Review by juzhiyuan — APPROVED** [review]
LGTM

**Comment by juzhiyuan** [comment]
Hi, these CIs are pending 🤔 Please be patient, and we will find out what happened.

**Comment by iamayushdas** [comment]
Ok please have a look. Thank you for support

**Comment by iamayushdas** [comment]
May i know what could be the problem here?

**Comment by juzhiyuan** [comment]
let me retrigger CI

**Pull request merged**